.github/workflows/ci.yml

```diff
@@ -234,11 +234,7 @@ jobs:
           miniforge-version: latest
           use-mamba: true
       - name: ASV installation
-        run: |
-          # FIXME: use the tag or release version of ASV as soon as it appears;
-          # The ability to build a conda environment by specifying yml file has not
-          # yet appeared in the release versions;
-          pip install git+https://github.com/airspeed-velocity/asv.git@ef016e233cb9a0b19d517135104f49e0a3c380e9
+        run: pip install asv==0.6.0
```

Comment:
please sync the changes and PR/issue title - this line installs `asv 0.6.0` but title claims `0.5.1` 🙃

Reply:
I just wanted to try the mamba plugin, it doesn't work yet, we have to wait for 0.6.1. Reverted changes to 0.5.1.